manim/mobject/geometry.py

```diff
@@ -1264,6 +1268,52 @@ def construct(self):
 
                 self.add(Group(g1, g2, g3).arrange(buff=2))
 
+
+    .. manim:: ArrowExample
+        :save_last_frame:
+
+        class ArrowExample(Scene):
+            def construct(self):
+                # LEFT Column: Varying Buff
+                v1 = VGroup()
```

Comment:
Instead of having a comment, a better named variable would be more appropriate.
```suggestion
                left_group = VGroup()
```